README.md

```diff
@@ -239,6 +199,17 @@ func (e *exporter) ExportView(vd *view.Data) {
 
 ```
 
+Configure the default interval between reports of collected data.
+This is a system wide interval and impacts all views. The default
+interval duration is 10 seconds. Trying to set an interval with
+a duration less than a certain minimum (maybe 1s) should have no effect.
```

Comment:
Currently there is no limit. Let's remove the  sentence about trying to set an internal with maybe 1s.